db/migrate/20240617105409_remove_oval_permissions.rb

```diff
@@ -0,0 +1,17 @@
+class RemoveOvalPermissions < ActiveRecord::Migration[6.1]
+  def up
+    Permission.where(resource_type: 'ForemanOpenscap::OvalPolicy').destroy_all
+    Permission.where(resource_type: 'ForemanOpenscap::OvalContent').destroy_all
+    Permission.where(resource_type: 'ForemanOpenscap::Cve').destroy_all
+
+    # Not a permission, but still a leftover from OVAL removal
+    if ForemanOpenscap.with_remote_execution?
+      rex_feature = RemoteExecutionFeature.find_by(label: 'foreman_openscap_run_oval_scans')
+      template_id = rex_feature.job_template_id
```

Comment:
Also, what if the user re-mapped the feature to a different template? Shouldn't we rather remove the one we seeded?

Reply:
I'd hope that no one did that. Even though they did, we won't support that in any way. I can imagine they might want to use that template to run some jobs manually with direct call to the `oscap` utility though... Thanks, updated.